certified-connectors/ZohoSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1943 @@
+                    },
+                    "org_name": {
+                      "type": "string",
+                      "description": "Displays the Organistaion Name"
+                    },
+                    "users": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "created_time": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Displays the time the document was created"
+                          },
+                          "user_status": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "user_status"
+                          },
+                          "user_email": {
+                            "type": "string",
+                            "description": "Shows the email id of the particular user"
+                          },
+                          "user_id": {
+                            "type": "string",
+                            "description": "Shows the user id of the particular user"
+                          },
+                          "last_name": {
+                            "type": "string",
+                            "description": "Displays the Last Name (if provided) of the user"
+                          },
+                          "ZUID": {
+                            "type": "string",
+                            "description": "Showcases the Zoho User ID"
+                          },
+                          "first_name": {
+                            "type": "string",
+                            "description": "Displays the First Name of the user"
+                          }
+                        },
+                        "description": "Shows the total number of users assosicated."
+                      }
+                    }
+                  },
+                  "description": "accounts"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "get-account-details"
+      }
+    },
+    "/users/invite": {
+      "post": {
+        "operationId": "invite-users",
+        "summary": "Invite the users whom you want to join",
+        "description": "Invite users with email id",
+        "consumes": [
+          "application/x-www-form-urlencoded"
+        ],
+        "parameters": [
+          {
+            "in": "formData",
+            "name": "email_id",
+            "type": "string",
+            "description": "[\"Enter the email ids of all the invitees whom you want to to be added\"]",
```

Comment:
@ZohoSign-integ Can you please let us knw why this description is double quotes

Reply:
It is to highlight the sentence inside the square brackets, for better understanding.